### PR TITLE
Instructor home page: do not auto-load response rate #6584

### LINKED ISSUE #6584: Instructor home page: do not auto-load response rate
Currently, response rate for some of the recent sessions are auto-loaded when the instructor visits the home page. This is an unnecessary strain on the server as most of these numbers may not even be read by the instructor. Let's not auto-load the response rate for any sessions. The instructor can click the the 'show' link to load the response rate if he wishes to see the number.
image

### PR DESCRIPTION
**Pull request by sznicci**
Fixes #6584 

I have removed the auto-load response rate and replaced it with the 'Show' link.

### CODE REVIEW

**Comment by sznicci** [comment]
This pull request is ready for review. :)

File: src/main/webapp/WEB-INF/tags/instructor/home/courseTable.tag [hunk 1]
```diff
@@ -48,7 +48,7 @@
                     ${sessionRow.status}
                 </span>
             </td>
-            <td class="session-response-for-test<c:if test="${sessionRow.recent}"> recent</c:if>">
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/instructor/home/courseTable.tag]: Check for the variable `isRecent` at `InstructorHomeFeedbackSessionRow`. If its sole usage is here (which is removed), then remove the variable as well.

**Comment by sznicci** [comment]
Thank you @wkurniawan07 for the review!
I've made the changes and the PR is ready for review again.

File: src/main/webapp/WEB-INF/tags/instructor/home/courseTable.tag [hunk 2]
```diff
@@ -48,7 +48,7 @@
...
+            <td class="session-response-for-test">
```
Comment (damithc) [inline src/main/webapp/WEB-INF/tags/instructor/home/courseTable.tag]: @wkurniawan07 the `-test` suffix is odd :-p

Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/instructor/home/courseTable.tag]: @damithc that's not any of the present dev's fault (#3601) and changing it here will cause unnecessary troubles.

Comment (damithc) [inline src/main/webapp/WEB-INF/tags/instructor/home/courseTable.tag]: It's fine if there is a separate issue for it.

**Pull request merged**